### PR TITLE
Release notes for edge-19.10.3

### PR DESCRIPTION
**Pull request by cpretzer**
Signed-off-by: Charles Pretzer <charles@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+* CLI
+    * Made `--cluster-domain` an install-only flag (thanks @bmcstdio!)
+    * Updated `check` to ensure that proxy trust anchors match configuration
+       (thanks @ereslibre!)
+* Proxy
+    * Added support for headless services (thanks @JohannesEH!)
+* Internal
+    * Added docker tag name validation for local development 
+       (thanks @srv-twry!)
```
Comment (olix0r) [inline CHANGES.md]: I would leave this out, as it's not user-facing.

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+This edge release adds support for headless services, improves the upgrade
+process after installing Linkerd with a custom cluster domain, and enhances
+the `check` functionality to report invalid trust anchors.
+
+* CLI
+    * Made `--cluster-domain` an install-only flag (thanks @bmcstdio!)
+    * Updated `check` to ensure that proxy trust anchors match configuration
+       (thanks @ereslibre!)
+* Proxy
+    * Added support for headless services (thanks @JohannesEH!)
```
Comment (olix0r) [inline CHANGES.md]: This was a controller change, not a proxy change.

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+* CLI
+  * Made `--cluster-domain` an install-only flag (thanks @bmcstdio!)
+  * Updated `check` to ensure that proxy trust anchors match configuration
+       (thanks @ereslibre!)
+* Controller
+  * Added support for headless services (thanks @JohannesEH!)
+* Helm
+  * Updated the helm build to retain previous releases
+
+Special thanks to @srv-twry for submitting bugfixes!
```
Comment (ihcsim) [inline CHANGES.md]: Is this referring to either 1e44513f30dadc36d9affd6077363c5fb9882f81 or d95a469a60236cf1ff9a62e23b24375a57ebd49e? I think we can either leave it out, or if it's referring to the CLI fix, then we can add something like to the CLI section:

```
Fixed an output sorting bug if the `metrics` command (thanks @srv-twry)
```

Comment (cpretzer) [reply CHANGES.md]: yeah, it's for [1e44513](https://github.com/linkerd/linkerd2/commit/1e44513f30dadc36d9affd6077363c5fb9882f81)

I'll pull it out

**Pull request merged**